models/exposure.py

```diff
@@ -115,6 +115,31 @@ def clear_cache(self):
 im_type_enum = Enum("science", "reference", "difference", "bias", "dark", "flat", name='image_type')
 
 
+def get_short_image_type(image_type):
```

Comment:
What do you see as the use case for this?  It's possible that adding this will just add the ability for confusion.  I think we'd be better off just defining which types exist (sci, ref, sub, etc.), and documenting that.  I see where this could add convenience, but it also adds new potential for "sometimes" errors (e.g. somebody calls get_short_image_type("subtraction") expecting it to return "sub").